Classes/OldModel/AbstractTimeSpan.php

```diff
@@ -305,4 +327,21 @@ public function getNumberOfPlaces(): int
      * @return string our places or an empty string if the timespan has no places
      */
     abstract public function getPlaceShort(): string;
+
+    /**
+     * Gets the hook provider for the list view.
```

Comment:
"for the list view" looks like a copy'n'paste error.

Reply:
Indeed, it is.